webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.js

```diff
@@ -12,9 +12,18 @@ class BreadcrumbBar extends React.Component {
       loadSwitcherResourcesByResource,
       currentPage,
       resourceUrl,
+      resourceSwitcherItems,
     } = this.props;
-
-    if (!currentPage || resourceUrl !== resource.resourceUrl) {
+    const isCorrectList =
+      resourceSwitcherItems.length > 0
```

Comment:
0 is a false value in js, so `length > 0` is redundant

Reply:
Changed